### PR TITLE
Create Login Page

### PR DESCRIPTION
**Pull request by caseyszilagyi**
## Feature Description (what did you do?)

Created a login page that hits the /api/v1/auth/token endpoint upon submission and stores the access token when received 

## Design/Implementation Details (how did you do it?)

Used the Primereact library to create a form with a textbox/submit button. Used the axios library to make the HTTP request.

Also deleted some default files from the template project

## Testing (how did you ensure this works correctly?)

Deployed locally, hit the endpoint and checked that the token was logged to the console

## Future Considerations (is there anything we should know about this going forward?)

Will need to do a refresh token in a future PR

### CODE REVIEW

**Review by hkder — DISMISSED** [review]
LGTM

File: frontend/src/pages/loginPage.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  // Hits the token endpoint, and stores the token in local storage. Displays incorrect password text if error returned from endpoint
+  onSubmit = (event: FormEvent<HTMLFormElement>): void => {
+    const reqOpts = {
+      url: process.env.REACT_APP_API_ENDPOINT_TOKEN,
+      headers: { "Content-Type": "application/json" },
+      method: "POST",
+      data: JSON.stringify({
+        email: "hk196@duke.edu",
+        password: this.state.password,
```
Comment (nex-pixel) [inline frontend/src/pages/loginPage.tsx:34]: Were you able to test for the password to be accepted or recieve a affirmative response

Comment (nex-pixel) [reply frontend/src/pages/loginPage.tsx:34]: Everything else for frontend looks good. I was able to run it, can't say anything for authentication itself without the backend people.

**Review by nex-pixel — CHANGES_REQUESTED** [review]

File: frontend/src/pages/loginPage.tsx (hunk 1 above)
Comment (caseyszilagyi) [reply frontend/src/pages/loginPage.tsx:34]: Yes, I checked by logging `response.data.access` (the token) to the console. I also checked that an incorrect password led to no token and an error message

**Review by nex-pixel — APPROVED** [review]
Everything looks good

**Review by nex-pixel — APPROVED** [review]

**Pull request merged**